### PR TITLE
Usage and Documentation updates

### PR DESCRIPTION
**Pull request by ranashreyas**
This branch contains an additional usage.md, containing a thorough walkthrough with screenshots and text, of how to pin the extension and use it.

### CODE REVIEW

File: src/background.js
Comment (ranashreyas) [inline src/background.js:10]: On install, it should open this usage link so the user has instructions on how to use it from the start. However, if tested right now without merging, this wouldn't work, as the main branch would not have the usage.md. If you want to see how it would look, look at the commented link.

File: usage.md
Comment (ranashreyas) [inline usage.md:2]: added file. Note: images like the logo are outdated, as the new logo is part of a new rebranding branch, and will work once merged.

**Review by ranashreyas — COMMENTED** [review]
@balajialg and @ryanlovett
Usage and Documentation review 1

**Review by ryanlovett — APPROVED** [review]
Having instructions appear after install sounds very useful.

@balajialg Any thoughts on the instructions? What about how it complements the page at https://ds-modules.github.io/curriculum-guide/technology/interact-links.html? Should the instructions link to the [troubleshooting](https://ds-modules.github.io/curriculum-guide/technology/jupyter/troubleshoot-nbgitpuller.html) page as well?

**Comment by balajialg** [comment]
@ryanlovett I guess linking to [troubleshooting page](https://ds-modules.github.io/curriculum-guide/technology/jupyter/troubleshoot-nbgitpuller.html) will be helpful in case users are having merge conflict issues. The other page is good to have as instructors might have that information by the time they install the extension.

**Comment by ryanlovett** [comment]
@balajialg Now that I think about it, I guess that could be a separate PR, if we want to go that way.

File: usage.md
Comment (balajialg) [inline usage.md:32]: @ranashreyas Do you want to fix usage.md updates as a separate PR or in the same PR?

**Review by balajialg — APPROVED** [review]

**Comment by balajialg** [comment]
@ryanlovett Yeah, we can unblock Shreyas from publishing the extension by scoping them for future PRs. We can keep suggesting incremental changes that he can fix going forward

**Pull request merged**
